errors/error.go

```diff
@@ -270,14 +301,47 @@ func (e *Error) error(verbose bool) string {
 	return strings.Join(errParts, separator)
 }
 
+func (e *Error) defaultErrorParts() []interface{} {
+	return []interface{}{
+		e.FileRange,
+		e.Kind,
+		e.Description,
+		e.Stack,
+		e.Err,
+	}
+}
+
 // Error returns the error message.
 func (e *Error) Error() string {
-	return e.error(false)
+	return e.error(e.defaultErrorParts(), false)
 }
 
 // Detailed returns a detailed error message.
 func (e *Error) Detailed() string {
-	return e.error(true)
+	return e.error(e.defaultErrorParts(), true)
+}
+
+// AsList returns the error as a list.
+// If it's underlying error is a *List, then it just returns it because
+// they're already explicitly wrapped.
+func (e *Error) AsList() *List {
+	if errs, ok := e.Err.(*List); ok {
```

Comment:
you're right.
I created this method to easily always handle errors as it it's a list, even if it's just one error, to avoid type switches or multiple `errors.As()` calls in downstream packages.